package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Champaign",
+  "description": "A web platform for online campaigning.",
+  "dependencies": {
+    "react": "^0.13.3",
+    "react-tools": "*",
+    "jest-cli": "0.4.5",
+    "browserify": "~> 10.2.4",
+    "browserify-incremental": "^3.0.1",
+    "babelify": "^6.1.3",
+    "reactify": "1.1.1"
```

Comment:
@osahyoun reactify is deprecated. babelify has JSX conversion built in, so we use that instead.